test/lib/ses_test.exs

```diff
@@ -136,6 +136,101 @@ defmodule ExAws.SESTest do
     end
   end
 
+  describe "#send_bulk_templated_email" do
+    test "with required params only" do
+      template = "my_template"
+      source = "user@example.com"
+
+      replacement_template_data1 = %{data1: "value1"} |> Poison.encode!()
```

Comment:
If those values will always be encoded, then I guess it makes sense to encode them inside a `send_bulk_templated_email/3`

Reply:
Yeah, good Idea!